main.css

```diff
@@ -92,10 +92,66 @@ body{
     font-weight: bold;
 }
 
+.functions--content {
+    background-color: #463431;
+    text-align: center;
+}
+
+.functions--content--container {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-evenly;
+    align-content: center;
+}
+
+.functions--content--container > div {
+    background-color: #f1f1f1;
+    margin: 1rem;
+    width: 45vw;
+    border: solid #b7922c;
+    border-radius: 40rem;
+}
+
+.functions--img {
+    width: 100%;
+    height: 100%;
+    max-width: 200px;
+    max-height: 200px;
+}
+
+.functions--content--header {
+    color: #b7922c;
+    margin-top: 4rem;
+}
+
+.functions--content--paragraph {
+    color: #845d44;
+    max-width: 50%;
+    margin: 1rem auto;
+}
+.test-img-container--1 {
+    width: 200px;
+    height: 200px;
+    background-image: url("./Ikony/kufer2.png");
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    margin: 0 auto;
+}
+
+.test-img-container--2 {
```

Comment:
test-img-container--1 and 2 is very similiar, please make a class with shareable styles

Reply:
two different containers were needed because of different icons used in each container - I assume I should create one class (test-img-container) containing repeated settings:
     width: 200px;
     height: 200px;
     background-position: center;
     background-repeat: no-repeat;
     background-size: cover;
     margin: 0 auto;

and leave test-img-container--1 and test-img-container--2 only with background-image: url("./Ikony/kufer2.png") and background-image: url("./Ikony/flgi.png") respectively?

with expected equally sized/scaled icons, need for these classes will be gone anyway - what to do in this situation?